src/components/masthead/masthead.js

```diff
@@ -185,14 +192,22 @@ class Masthead extends React.Component {
         <Toolbar>
           <ToolbarGroup className={css(accessibleStyles.screenReader, accessibleStyles.visibleOnLg)}>
             <ToolbarItem className={css(accessibleStyles.screenReader, accessibleStyles.visibleOnMd)}>
-              <Button
-                className="pf-c-button pf-m-plain"
-                aria-label="Settings"
-                variant="plain"
-                onClick={this.onSettingsClick}
-              >
-                <CogIcon />
-              </Button>
+              {isAdmin ? (
+                <Button
+                  className="pf-c-button pf-m-plain"
+                  aria-label="Settings"
+                  variant="plain"
+                  onClick={this.onSettingsClick}
+                >
+                  <CogIcon />
+                </Button>
+              ) : (
+                <Tooltip position={TooltipPosition.bottom} distance="30" content={<div>{settingsTooltip}</div>}>
```

Comment:
The delay of the tooltip caused me to initially click the cog icon, but when you click on it, the tooltip won't show up again until you mouse away from it and then hover over the cog. I think setting entryDelay={0} on Tooltip could mitigate this since the tooltip will just show up immediately. What do you guys think?